src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -179,147 +242,1364 @@ resource linkedService_storageAccount 'Microsoft.DataFactory/factories/linkedser
+          {
+            activity: 'Get Config'
+            dependencyConditions: [
+              'Succeeded'
+            ]
+          }
+        ]
+        userProperties: []
+        typeProperties: {
+          items: {
+            value: '@activity(\'Get Config\').output.firstRow.exportScopes'
+            type: 'Expression'
+          }
+          isSequential: true
+          activities: [
+            {
+              name: 'Get exports for scope'
+              type: 'WebActivity'
+              dependsOn: []
+              policy: {
+                timeout: '0.00:05:00'
+                retry: 2
+                retryIntervalInSeconds: 30
+                secureOutput: false
+                secureInput: false
+              }
+              userProperties: []
+              typeProperties: {
+                url: {
+                  value: '@{variables(\'resourceManagementUri\')}@{item().scope}/providers/Microsoft.CostManagement/exports?api-version=2023-03-01'
+                  type: 'Expression'
+                }
+                method: 'GET'
+                authentication: {
+                  type: 'MSI'
+                  resource: {
+                    value: '@variables(\'resourceManagementUri\')'
+                    type: 'Expression'
+                  }
+                }
+              }
+            }
+            {
+              name: 'Run exports for scope'
+              type: 'ExecutePipeline'
+              dependsOn: [
+                {
+                  activity: 'Get exports for scope'
+                  dependencyConditions: [
+                    'Succeeded'
+                  ]
+                }
+              ]
+              userProperties: []
+              typeProperties: {
+                pipeline: {
+                  referenceName: pipeline_run.name
+                  type: 'PipelineReference'
+                }
+                waitOnCompletion: true
+                parameters: {
+                  ExportScopes: {
+                    value: '@activity(\'Get exports for scope\').output.value'
+                    type: 'Expression'
+                  }
+                  Recurrence: {
+                    value: '@pipeline().parameters.Recurrence'
+                    type: 'Expression'
+                  }
+                }
+              }
+            }
+          ]
+        }
+      }
+    ]
+    concurrency: 1
     parameters: {
+      Recurrence: {
+        type: 'string'
+        defaultValue: 'Daily'
+      }
+    }
+    variables: {
       fileName: {
         type: 'String'
+        defaultValue: 'settings.json'
       }
       folderName: {
         type: 'String'
+        defaultValue: 'config'
+      }
+      finOpsHub: {
+        type: 'String'
+        defaultValue: hubName
+      }
+      resourceManagementUri: {
+        type: 'String'
+        defaultValue: environment().resourceManager
       }
-    }
-    type: any(convertToParquet ? 'Parquet' : 'DelimitedText')
```

Comment:
If we're forcing parquet, we should get rid of the `convertToParquet` parameter.